ml_models/use_model.py

```diff
@@ -63,10 +69,29 @@ def get_answer(query):
     ans_list = [get_text_by_ind(ind) for k, ind in indexer.return_closest(query, k=4)]
     return ans_list
 
+
 def get_answer_ind(query):
     ind_list = [get_new_ind_by_ind(ind) for k, ind in indexer.return_closest(query, k=4)]
     return ind_list
 
+
+def get_slack_answer_ind(query, k=4):
+    import requests
```

Comment:
Import a library for each query call

Reply:
I suggest to move this code in a new script

Reply:
done